api/src/test/java/org/openmrs/ConceptNumericTest.java

```diff
@@ -122,6 +122,7 @@ public void shouldChangeConceptMapReferenceToParentConcept() throws Exception {
 	public void shouldSaveAConceptNumericWithAllowDecimalValue() throws Exception {
 		Concept c = Context.getConceptService().getConcept(22);
 		ConceptNumeric cn = new ConceptNumeric(c);
+		cn.addDescription(new ConceptDescription("some description", Context.getLocale()));
```

Comment:
Alternatively, i think it should be okay for one to pass in a null locale and then in ConceptSaveHandler you set it to what is returned to Context.getLocale(), we could also add a constructor that only takes a actual description and the locale would still get set by the save handler